src/int/i15_montmul.c

```diff
@@ -136,9 +136,167 @@ loop%=:                                                            \n\
 		v = len4;
 #else
 		r = 0;
+#ifdef ESP8266
```

Comment:
Can you please drop the ifdef esp8266 and use this for all compilations?  Since there is no assembly here, it should work on any system.

I run the BearSSL self test suite on the host which lets me find when I break something logically (doesn't catch misaligned reads, but will catch if the actual values read are wrong due to shifts, read size problems, etc.)

Reply:
Sure